upstream.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = xxHash-0.8.0
+
+source_url = https://github.com/Cyan4973/xxHash/archive/v0.8.0.tar.gz
+source_filename = xxHash-0.8.0.tar.gz
+source_hash = 7054c3ebd169c97b64a92d7b994ab63c70dd53a06974f1f630ab782c28db0f4f
+
+[provides]
```

Comment:
This should be `provide`, not `provides`.

Also, all files are indented with tabs. This is prohibited, only spaces are allowed.

Reply:
Noted. Are tabs not allowed because Meson doesn't support them well or because it is just the style guide? I have been using tabs in my Meson projects so if there is a functional reason to not use tabs, I definitely want to fix that

Reply:
The general style guide of Meson is "No tabs ever". There is no functional reason. I considered making tabs a hard error when writing the parser but chose against it because it would probably have lead to a fair bit of backlash.

Reply:
That's fine. I'll change up my project files to use spaces instead.

Also I noticed that the descriptions look like shit in Meson configure output? Any tips on trying to make them look a little decent? I know they are very long, but I just pulled them from upstream.

Reply:
"too-long descriptions won't be read by anyone" isn't the worst policy to take, most of the wording there is not really helpful in help output IMO.

Reply:
I have changed the option descriptions to just `See XXH_$buildoption`. In the xxHash repo, the README has great descriptions of all the options.